e2e/volume_ops_test.go

```diff
@@ -271,6 +271,16 @@ func testVolumeDelete(t *testing.T) {
 func testVolumeStart(t *testing.T) {
 	r := require.New(t)
 	r.Nil(client.VolumeStart(volname, false), "volume start failed")
+
+	r.Nil(client.VolumeStart(volname, true), "volume force start failed")
+	vol, err := client.BricksStatus(volname)
+	r.Nil(err)
+	for index := range vol {
+		if vol[index].Port == 0 || !vol[index].Online {
+			r.Nil(errors.New("volume force start failed"))
```

Comment:
You don't have to do simply create an error so that to use `r.Nil()`. You can use `r.NotZero` for port and r.True` for Online.

See https://godoc.org/github.com/stretchr/testify/require for available functions